### DISCUSSION TITLE
有什么办法扩展 @formily/antd 的某个组件吗？

### DISCUSSION THREAD

**Discussion by xuanhen2013** [alibaba/formily#3326] [Q&A 2.x] [answered]
场景：
目前 @formily/antd 的 ArrayTable 所有按钮都为 Icon，想自定义为文字并且增加自定义点击逻辑，但是其他的组件依然保留。

**Comment by janryWang** [comment, marked as answer]
用ArrayTable提供的useIndex这种hook，自定义组件，然后schema定义的时候直接指明你的组件即可
